packages/components/src/components/form/Select/index.tsx

```diff
@@ -49,18 +54,30 @@ const selectStyle = (
             },
         };
     },
+    valueContainer: (base: Record<string, any>) =>
+        ({
+            ...base,
+            border: 0,
+            padding: isClean ? '0 3px 0 0' : '2px 8px',
+            fontWeight: isClean ? variables.FONT_WEIGHT.MEDIUM : variables.FONT_WEIGHT.REGULAR,
+            minWidth,
+            display: 'flex',
+            flexWrap: 'nowrap',
+            justifyContent: isClean ? 'flex-end' : 'flex-start',
+        } as const),
     indicatorSeparator: () => ({
         display: 'none',
     }),
     dropdownIndicator: (base: Record<string, any>, { isDisabled }: { isDisabled: boolean }) => ({
         ...base,
         display: !withDropdownIndicator || isDisabled ? 'none' : 'flex',
```

Comment:
CleanSelect variant had another "feature", dropdown would show only on hover/focus. But I didn't like it anyway so I am okay with omitting it. 🤐